core/frontend/apps/amp/lib/views/amp.hbs

```diff
@@ -1,23 +1,573 @@
+    <style amp-custom>
```

Comment:
Wondering, if we should minimise the CSS again 🤔

Reply:
Don't think it's worth it! I believe google compress AMP files anyway when they cache them on their own CDN, and this is way more maintainable / simple, with basically no difference in filesize cause of how little code it is :)